node/api/renter.go

```diff
@@ -356,24 +356,21 @@ func (api *API) renterDownloadHandler(w http.ResponseWriter, req *http.Request,
 		return
 	}
 
-	if params.Async { // Create goroutine if `async` param set.
-		// check for errors for 5 seconds to catch validation errors (no file with
-		// that path, invalid parameters, insufficient hosts, etc)
-		errchan := make(chan error)
-		go func() {
-			errchan <- api.renter.Download(params)
-		}()
+	errChan := api.renter.Download(params)
```

Comment:
Again, not sure I consider this change to be an improvement over the original.

Reply:
See the explanation above. It also gets rid of the hardcoded 100ms latency for async download calls.